tasks/main.yml

```diff
@@ -33,6 +43,32 @@
       when:
         - kernel_settings_transactional_update_reboot_ok is none
 
+- name: Get package_facts for tuned info
+  package_facts:
+
+- name: Set tuned version
+  set_fact:
+    __kernel_settings_tuned_version: "{{
+      ansible_facts.packages['tuned'][0]['version'] }}"
+
+- name: Read tuned main config
+  kernel_settings_get_config:
+    path: "{{ __kernel_settings_tuned_main_conf_file }}"
+  register: __kernel_settings_register_tuned_main
+
+# this is the parent directory for the profile sub-directories
+- name: Set tuned profile parent dir
+  set_fact:
+    __kernel_settings_profile_parent: "{{ __prof_from_conf
+      if __prof_from_conf | length > 0 else
+      __kernel_settings_tuned_dir ~ __prof_subdir }}"
+  vars:
+    __data: "{{ __kernel_settings_register_tuned_main.data }}"
+    __prof_from_conf: "{{ __data.get('profile_dirs', '').split(',')[-1] }}"
+    __prof_subdir: "{{ '/profiles'
+      if __kernel_settings_tuned_version is version('2.23.0', '>=')
+      else '' }}"
```

Comment:
You can set __kernel_settings_tuned_version here too, it's not used anywhere else